rmw_fastrtps_cpp/src/functions.cpp

```diff
@@ -618,13 +618,18 @@ rmw_ret_t rmw_init()
   return RMW_RET_OK;
 }
 
-rmw_node_t * rmw_create_node(const char * name, size_t domain_id)
+rmw_node_t * rmw_create_node(const char * name, const char * namespace_, size_t domain_id)
 {
   if (!name) {
     RMW_SET_ERROR_MSG("name is null");
     return NULL;
   }
 
+  if (!namespace_) {
```

Comment:
Yeah, my assumption is that it is a valid string at this point. See a test for this in `rcl`:

https://github.com/ros2/rcl/pull/112/files#diff-56a2b9e98ab73d202a5f841102b7c54bR326